example/package.json

```diff
@@ -13,20 +13,20 @@
     "@react-navigation/core": "../",
     "@react-navigation/native": "^3.1.4",
     "expo": "~32.0.0",
-    "hoist-non-react-statics": "^2.5.0",
+    "hoist-non-react-statics": "^3.3.0",
     "prop-types": "^15.6.0",
     "query-string": "^6.2.0",
-    "react": "16.5.0",
-    "react-is": "^16.5.2",
+    "react": "16.8.3",
```

Comment:
We can't bump React version here without matching Expo/RN version

Reply:
This is what I meant on the earlier PR and in Discord -- can the example not be tied to expo for this reason? We consume react-navigation outside of Expo (even though Expo is fantastic and a good choice for many), and are in a situation where waiting for expo to catch up to the new RN isn't really viable.

I can change the example to use react-navigation-web as an e2e example, even though that means there will be two non-atomic commits that need to be wrangled across repo boundaries.

Reply:
> We consume react-navigation outside of Expo (even though Expo is fantastic and a good choice for many), and are in a situation where waiting for expo to catch up to the new RN isn't really viable.

We can safely use `React.createContext` without having to upgrade React or Expo. It's been in React for quite a while. IMO it doesn't really matter what version of React Native or Expo the example uses. Changing React/RN version should be unnecessary to drop create-react-context.

> can the example not be tied to expo for this reason?

We should also keep in mind that we can't depend on the latest React/React Native version since the library needs to support older React Native versions. Expo is just 1-2 versions behind React Native most of the time. I think it's not practical to drop support for RN versions this old.

Reply:
that's totally fair, and you're correct about when the API went public. I was confused because two other repos removed the dependency when they bumped to React 16.7 and I assumed that particular part of context API became available then. I removed those upgrade parts and the PR is much simpler now. Thanks!